### ISSUE TITLE
Hyperimidodipeptiduria 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6226]
**Preferred term label:**
Hyperimidodipeptiduria 

**Synonyms**
Imidodipeptiduria 

**Definition (free text, please give PubMed ID)**
Elevated levels of proline-containing dipeptides in urine.
PMID:9196362

Comment: Prolidase splits dipeptides containing C-terminal proline or hydroxyproline. In prolidase deficiency, normal recycling of proline cannot occur, and this leads to iminopeptiduria, the characteristic biochemical abnormality. Glycylproline is the major urinary imidodipeptide. Leucylproline can also be identified in persons with prolidase deficiency.

**Parent term (use hpo.jax.org/app)**
consider new Peptiduria term

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/170100

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
